src/Page/src/ConfigProvider.php

```diff
@@ -26,12 +30,14 @@ public function getDependencies(): array
         return [
             'delegators' => [
                 Application::class => [
-                    RoutesDelegator::class,
+                    AppRoutesDelegator::class,
```

Comment:
`Light\App\RoutesDelegator` should be delegated from `App\ConfigProvider`.
Only `Light\Page\RoutesDelegator` should be in this file.